### DISCUSSION TITLE
I suggest the untranslated part of Korean.

### DISCUSSION THREAD

**Discussion by ghost** [qnblackcat/uYouPlus#411] [Ideas]
I am reporting that there is a part that is not translated into Korean. I hope you will accept this idea.

< Setting -> General >

Enable vote submission -> 투표 제출 사용

Allow your unique anonymous ID and videos/shorts likes/ dislikes data to be submitted to returnyoutubedislikeapi.com

-> 당신의 익명 ID 및 동영상/쇼츠 좋아요/싫어요 관련 데이터를 returnyoutubedislikeapi.com에 제출하는 것을 허용합니다.

< YouPiP > 

Use PiP Button > PiP 버튼 사용
- Adds a PiP button over the video control overlay to activate PiP instead of dismissing the app.
(영상 제어 오버레이 위에 PiP 버튼을 직접 추가하여 기존 PiP 방식과는 다른 방법으로 PiP를 활성화합니다.)

Use Video Tab Bar PiP Button > 비디오 툴바 PiP 버튼 사용
- Adds a PiP button in video tab bar to activate PiP instead of dismissing the app. App restart is required.
(앱을 해제하는 대신 PiP를 활성화하기 위해 영상 탭 표시줄에 PiP 버튼을 추가합니다. 앱을 다시 시작해야 합니다.)

Disable PiP for Mini Player -> 미니플레이어 PiP 비활성화
- Disables PiP while playing a video in the mini player.
(미니 플레이어에서 영상을 재생하고 있는동안 PiP를 비활성화 합니다.)

Legacy PiP -> 레거시 PiP
- Uses AVPlayerLayer for PiP. This fixes speedup bug but also removes UHD options (2K/4K) from any videos. App restart is required.
(PiP용 AVPlayerLayer를 사용합니다. 이것은 속도향상 버그를 수정하지만 모든 영상에서 UHD 옵션(2K/4K)이 제거됩니다. 앱 재시작이 필요합니다.)

Non-backgroundable PiP -> 강제 백그라운드 PiP
- Enables PiP for non-backgroundable video.
(백그라운드가 되지 않는 영상에 PiP를 활성화합니다.)

Fake YouTube version -> 가짜 유튜브 버전
- Set YouTube version to 16.46.5 so that PiP button under video player may show.
(영상 플레이어 아래에 PiP 버튼이 나타날 수 있도록 YouTube 버전을 16.46.5로 설정합니다.)

**Comment by qnblackcat** [comment]
Feel free to submit Korean localization for YouPiP here:
https://github.com/PoomSmart/YouPiP/tree/main/layout/Library/Application%20Support/YouPiP.bundle
